src/connection/index.js

```diff
@@ -53,6 +53,33 @@ class Connection {
     );
   }
 
+  static async openAndUnlockAccounts(
```

Comment:
1. `open()` also unlocks the accounts. I would prefer to update `open()` instead of adding this function.
2. I think the passwords for origin and auxiliary should be able to differ. As argued above, the passwords should be provided from the outer CLI layer if user input is required.